### DISCUSSION TITLE
How to get over stairs?

### DISCUSSION THREAD

**Discussion by Yakov49** [ORelio/Minecraft-Console-Client#2406] [Q&A]
My bot cannot enter the portal, in the server lobby, because in order to enter it, you need to go through the steps, but the bot cannot do this, it says that it cannot go in this direction, what should I do? 
![image](https://user-images.githubusercontent.com/122409067/211679398-18aa6fd5-fb43-4660-a2ed-fbd3b3850eab.png)
Tips write that you need to enter -f, but when I write coordinates and -f and press enter, the command will simply disappear and the bot will not move! Please, help.

**Comment by BruceChenQAQ** [comment]
This may be a pathfinding-related bug, which may not be fixed in the near future due to some higher priority goals currently outstanding.

You can try turning off gravity handling, sending upward movement commands and forward (using East or West or South or North) movement commands.
